src/components/cards/CardBadges.vue

```diff
@@ -27,6 +27,7 @@
 		<div v-if="card.description && checkListCount > 0" class="card-tasks icon icon-checkmark">
 			{{ checkListCheckedCount }}/{{ checkListCount }}
 		</div>
+		<div v-else-if="card.description && checkListCount == 0" class="icon icon-description" />
```

Comment:
```suggestion
		<div v-else-if="card.description.trim() && checkListCount == 0" class="icon icon-description" />
```

To make sure that a description only containing spaces is not showing up.